pkg/plugins/common/kustomize/v2/scaffolds/internal/templates/config/rbac/role.go

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
It is not under the scope of this PR
The fix done in https://github.com/kubernetes-sigs/kubebuilder/pull/3626 is here otherwise that fails with the grafana sample